packages/relay/src/lib/errors/MirrorNodeClientError.ts

```diff
@@ -31,6 +31,7 @@ export class MirrorNodeClientError extends Error {
 
     static statusCodes = {
       NOT_FOUND: 404,
+      TOO_MANY_REQUESTS: 429,
       NO_CONTENT: 204
```

Comment:
also, should we add `400` error for `CONTRACT_REVERT_EXECUTED`

Reply:
No since it only applies to the contracts/call endpoint. 400 status code on other mirror node calls just means an error right.